### DISCUSSION TITLE
POLL: Using Callbook's QTH information for Digi modes (FT8 etc).

### DISCUSSION THREAD

**Discussion by foldynl** [foldynl/QLog#420] [Polls]
Currently, QLog does not use QTH information from the Callbook for digital modes (FT8, FT4, etc.). My rationale is based on issues #403 and #405. However, there are also votes for QLog to use the QTH information from the Callbook in the case of digital modes with incomplete gridsqure. 

So, I'd like to let the community make the decision.

Poll ends: 23/8/2024
Poll will be valid if 11 or more people respond.

**Comment by muihlinn** [comment]
I'm not specially interested in maintaining anything else but operator's name in case of callsign and gridsquare matching. 
All the other other stuff which comes with the QSL is enough to me.  Perhaps a checkbox in preferences to keep or not the data taken from the callbook is enough to make everyone happy. 

I'll vote yes because I'm between the two positions and having the choice seems better.

**Reply by foldynl** [reply]
QLog's goal is not to have a checkbox or other settings for everything. The aim is to fine-tune it so that it works without various user adjustments. It's always necessary to ask: Can't we obtain that information in another way? Is it really necessary to change this parameter? Therefore, don't think that just because we can't agree, a checkbox needs to be introduced.

**Reply by muihlinn** [reply]
Let me clarify. i'm not saying that qlog needs anything, in fact I don't use many of its features. I am using QLog because I was looking for a fairly minimal log program with some goodies, like cluster spotting. As I don't contest most features from other programs are dead weight to me.

On this particular question I raised the issue because after 2 or 3 months without antennas I compiled the then current version and it filled some data from callbook. The following version didn't, so I notified along other minor stuff as I did before. 

For what I've seen there are two opinions weighting about the same which won't/can't agree. A way to enable or disable this behavior seems reasonable in such case but that doesn't mean that I think it *needs* it. Personally I lean to fill callsign's personal data (operator name, address, email...), but I won't  worry if not.

**Comment by aa5sh** [comment]
I like the idea of the callbook data filling in anything that wasn't provided form WSJT-X or whichever digital mode app.  I could see not loading Gridsqaure or other information in the UDP blast.  But I wouldn't see harm in filling in Name, Email, URL etc. only if it wasn't supplied from the importing contact.

**Comment by dl2ki** [comment]
I tend to think that only unique data should be automatically transferred from callbooks. QRZ.com does not provide a grid square by default, so a comparison does not seem possible here. HamQTH provides a grid square, but it is often the case that the QTH information is different between QRZ.com and HamQTH. Apparently many users keep their QRZ.com pages more up to date. 

Personally, I check the data after each QSO based on the location of the station transmitted in the QSO. This is essential, especially for portable stations.

On the other hand, an 'ambiguous' data transmission for digital modes would not bother those who do not use digital modes.

**Comment by foldynl** [comment]
The community chose the variant 1 -  `Yes - QLog should use the QTH Info in case of matching callsign and partially matching the gridsquare`
